### DISCUSSION TITLE
#_EVENTIMAGETHUMBURL{Large} Not Working After Latest Update

### DISCUSSION THREAD

**Discussion by leonrainbow** [liedekef/events-made-easy#129] [General]
We really love this plugin! When we upgraded from version 2.3.48 to version 2.3.61

We were using the code below to pull the featured image size large:

[eme_if tag='#_EVENTIMAGETHUMBURL{large}']<img src="#_EVENTIMAGETHUMBURL{large}" class="img-fluid">[/eme_if]
[eme_if tag='#_EVENTIMAGETHUMBURL{large}' is_empty=1]<img src="/wp-content/themes/your-site/images/placeholder-1200.jpg" class="img-fluid">[/eme_if]

After the update, it was pulling the featured image thumbnail 150 x 150 . Luckily we had a backup of the plugin and we reverted back to it. We just wanted to bring it to your attention.
Thanks, Leon

**Comment by liedekef** [comment]
This should be a bug request, not a discussion :-)
Anyway, fixed here:
https://github.com/liedekef/events-made-easy/commit/c11ac30345d21be35e766fc0788bd9c262c81292

I'll release a fix-version today

**Reply by leonrainbow** [reply]
Thanks, I appreciate you!

**Reply by liedekef** [reply]
I released an update today (I released one yesterday but discovered another error in it ...)
